app/models/post_file.rb

```diff
@@ -0,0 +1,44 @@
+class PostFile < ActiveRecord::Base
+  belongs_to :post
+
+  validates :filename, presence: true
+  validates_associated :post
+
+  before_save :update_post
+
+  def metadata
+    @metadata ||= YAML.load_file(filename) || {}
+  end
+
+  def data
+    {
+      title: metadata['title'],
+      tag_list: (metadata['tags'] || '').downcase,
+      author: author,
+      body: body
+    }
+  end
+
+  def author
+    User.from_full_name(metadata['author'])
```

Comment:
I thought including support for both, but given that we're in the middle of changing our emails (we still use groupbuddies emails in several places, including our blog accounts) I thought it would introduce unnecessary problems.

It'll be done once we have fully migrated to the new emails